action/kafkaFeed.js

```diff
@@ -17,6 +17,11 @@ function main(params) {
     var namespace = encodeURIComponent(triggerComponents[1]);
     var trigger = encodeURIComponent(triggerComponents[2]);
 
+    if (namespace === '_') {
+        whisk.error('You must supply a non-default namespace.');
```

Comment:
the feed runs under the subject's namespace, so using the new properties is goodness. +1 @csantanapr's recommendation.